app.js

```diff
@@ -49,3 +49,19 @@ async function renderScenarios(data) {
 
 document.querySelector("button").onclick = () =>
```

Comment:
We don't need this one, this can be removed as it was just a temporary fix to let Gabriela work on her function

Reply:
Your function correctly places this line inside of your function

Reply:
We should also add a way to initialize the function when the page loads so that the html is rendered.  
I forgot to mention this in the instructions.

One final step we could take is to remove all html inside of the `<main>` container on the html file as our javascript is now configured to render it we just need to call it on pageload, so like `document.body.onload = appInit` would wait for the page to load and then initialize the app